packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_CoarseOperator_decl.hpp

```diff
@@ -132,12 +139,14 @@ namespace FROSch {
         MapPtrVecPtr GatheringMaps_;
         MapPtr CoarseSolveMap_;
         MapPtr CoarseSolveRepeatedMap_;
+        GOVecPtr BlockCoarseSize_;
```

Comment:
What is `BlockCoarseSize_`?

Reply:
It's the dimension of coarse space for a block (number of interface components)->Changed name to BlockCoarseDimension_.